evoseal/utils/validator.py

```
@@ -1,651 +1,695 @@
+                    message="Failed to load schema validator",
+                    code="schema_validation_error",
+                )
+                return False
+
         try:
-            # Always validate against schema first
             self.validator.validate(workflow)
-        except ValidationError as e:
-            path = "." + ".".join(str(p) for p in e.absolute_path) if e.absolute_path else ""
+            return True
+        except JSONSchemaValidationError as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)